src/content/components/AllocationView/AllocationView.js

```diff
@@ -0,0 +1,277 @@
+import React from "react";
+import styles from "./AllocationView.scss";
+import { GlobalContext } from "../GlobalContext/GlobalContext";
+import { BugList } from "../BugList/BugList";
+import { Loader, MiniLoader } from "../Loader/Loader";
+import { Container } from "../ui/Container/Container";
+import { BUGZILLA_TRIAGE_COMPONENTS } from "../../../config/project_settings";
+import { teams } from "../../../config/people";
+import { columnTransforms } from "../BugList/columnTransforms";
+
+// Add a view that includes an individual BugList for each OMC engineer,
+// tracking all unresolved bugs in our components that are assigned to that
+// engineer, have an iteration matching the current release, and have the points
+// field set. Display columns should include ticket, bug id, type, title,
+// iteration, priority, and points, in that order. Bugs should be sorted by
+// points first, then by priority. In other words, 7-pointers go before
+// 3-pointers, but if there are two 7-pointers, then the P1 should go before the
+// P2.
```

Comment:
These longer notes are just for development to make the spec easier to find. I'll remove em when we're done